systemtests/CMakeLists.txt

```diff
@@ -96,7 +96,7 @@ set(logdir "log")
 set(mon_dir_password "mon_dir_password")
 set(mon_fd_password "mon_fd_password")
 set(mon_sd_password "mon_sd_password")
-set(plugindir ${PROJECT_BINARY_DIR}/plugindir)
+set(plugindir ${CMAKE_SOURCE_DIR}/core/src/plugins)
```

Comment:
Is this change required? Is plugindir used afterwards?
I agree, that this setting is better than the setting before. However, it is still wrong, as ${CMAKE_SOURCE_DIR}/core/src/plugins) do not contain the plugins, but subdirectories for each daemon (dird, stored, filed) containing the plugins.

Reply:
The plugindir variable is currently handled very inconsistently everywhere. An alternative would be to create a plugindir in the test and link all files of the subdirectories (dird, filed etc.) with cmake.